repos/keg-components/src/components/form/scrollable/item/item.js

```diff
@@ -0,0 +1,58 @@
+import React, { useCallback } from 'react'
+import { Button } from 'KegButton'
+import { reStyle } from '@keg-hub/re-theme/reStyle'
+import { noOp } from '@keg-hub/jsutils'
+import PropTypes from 'prop-types'
+
+/**
+ * A Button styled for the autocomplete
+ */
+const SelectButton = reStyle(
```

Comment:
Seems like we should allow the props.styles to override these, unless I'm missing something?
Simple way to do that, is just do a `deepMerge` with `props.styles` and the final object on line 29.
We should  still be able to override them by passing it to the final component, but I have not verified that. If that's the case, then this is fine.

Reply:
Yep the `deepMerge` is done automatically within retheme. I updated the `item.example.js` story with custom styles to demonstrate this.